"0","load('./F03_SelPredictores/F03_4_ga_100.RData')
"
"2","Error in readChar(con, 5L, useBytes = TRUE) : cannot open the connection
"
